src/components/WrappedActionMenu/RewardsTab/RewardsTab.tsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react'
 import { Divider, Grid } from '@material-ui/core'
+import { displayDate } from '@consts/utils'
 import { RewardsLine } from '@components/WrappedActionMenu/RewardsTab/RewardsLine/RewardsLine'
 import { OutlinedButton } from '@components/OutlinedButton/OutlinedButton'
 import { RewardsAmount } from '@components/WrappedActionMenu/RewardsTab/RewardsAmount/RewardsAmount'
 import BN from 'bn.js'
 import useStyles from './style'
 
+export type RoundType = 'next' | 'current' | 'finished'
+
+export type RoundData = {
+  [type in RoundType]: {
+    roundPoints: BN
+    roundAllPoints: BN
+    roundStartSlot: BN
+  }
+}
+
 export interface IRewardsProps {
-  slot?: BN
+  slot: number
```

Comment:
why?

Reply:
Because Solana method that fetching the current slot returns `Promise<number>`